phpBB/styles/prosilver/theme/icons.css

```diff
@@ -85,3 +85,26 @@
     line-height: .9;
     height: 12px;
 }
+
+
+blockquote cite:before, .uncited:before {
+	-webkit-font-smoothing: antialiased;
+	-mos-osx-font-smoothing: grayscale;
+	font-family: FontAwesome;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	font-variant: normal;
+	line-height: 1;
+	text-rendering: auto;
+	display: inline-block;
+	padding-right: 5px;
+	content: '\f10d'; /* Font Awesome quote-left */
+}
+
+.rtl blockquote cite:before, .rtl .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
yes, also instead of retyping everything for the icon coding, just add `blockquote cite:before, .uncited:before` as more selectors in icon.css at the top like so

```css
 .icon, 
.button .icon,
blockquote cite:before, 
.uncited:before  {
 	display: inline-block;
   	font-weight: normal;
   	font-style: normal;
   	font-variant: normal;
   	font-family: FontAwesome;
   	font-size: 14px;
   	line-height: 1;
   	text-rendering: auto; /* optimizelegibility throws things off #1094 */
   	-webkit-font-smoothing: antialiased;
   	-moz-osx-font-smoothing: grayscale;
 }
```

This way the only thing added to content.css is the `padding-right`